suite/tests/client-interface/mangle_suspend.c

```diff
@@ -34,6 +34,11 @@
  * by asynch interrupt.
  */
 
+/* clang-format off */
+/* XXX: clang-format incorrectly detected a tab difference at "clang-format on"
+ * below. This is why "clang-format off" has been moved outside the ifdef until
+ * bug is fixed.
```

Comment:
nit: the bug

Also, if this is a bug in clang-format-diff and this comment expects it to be fixed, presumably it is filed somewhere (else it would never be fixed?) and we can cite that here?

Reply:
Yes, I can file this. Since the source file of this test is open source, I can link/attach this I assume?

Reply:
Yes, this is public.

Reply:
Merging now and filing clang bug shortly.